src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
+$bodyFont: 'Roboto';
+
+@import './header';
+@import './card';
+@import './starts';
```

Comment:
It seems like there is a typo in the file name 'starts'. If it is supposed to be 'stars', please correct the filename and the import statement to match the intended meaning as per the checklist item 'File names should be understandable and consistent'.